variables.tf

```
@@ -10,5 +10,36 @@ variable "location" {
 
 variable "CLIENT_ID" {
     description = "The Github Client ID for the Terraform Azure app"
-    default = "2849db5f59c50e93a017"
+}
+
+variable "CLIENT_SECRET" {
+    description = "The Github Client Secret for the Terraform Azure app"
+}
```

It would be worthwhile marking this as `sensitive` as well, to help avoid it being leaked by the console, e.g.

```suggestion
variable "CLIENT_SECRET" {
    sensitive   = true
    description = "The Github Client Secret for the Terraform Azure app"
}
```